### ISSUE TITLE
Show performance on main page

### ISSUE THREAD

**Issue by nightrome** [kazuto1011/deeplab-pytorch#2]
I am the author of COCO-Stuff and I must say this is pretty cool **stuff**!
Do you mind showing the validation set performance (mean IOU) on this GitHub page?
It would be interesting to compare to the performance I achieved in the arXiv paper.
Furthermore, in a few days we will publish the new COCO-Stuff with 164K images.
Would be cool if we could then update this code accordingly.

**Comment by kazuto1011** [comment]
Sorry for the late reply. Thank you for commenting on my implementation.
I have fixed some bugs in training procedure, although I haven't updated this repo.
The performance after 15k iterations has achieved 33.2 % mIoU on the validation set (I couldn't get 20k result because the process died unexpectedly).
I will update this page with the validation scores sometime soon!

**Comment by kazuto1011** [comment]
Updated ```README.md```.
Please see [this section](https://github.com/kazuto1011/deeplab-pytorch/blob/master/README.md#results) for a validation score (a little bit lower).
